src/workerd/io/compatibility-date.capnp

```diff
@@ -692,4 +692,9 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $experimental
       $neededByFl;
   # Enables cache settings specified request in fetch api cf object to override cache rules. (only for user owned or grey-clouded sites)
+
+  jsWeakRef @73 :Bool
+      $compatEnableFlag("enable_weak_ref")
+      $compatDisableFlag("disable_weak_ref");
+  # Enables WeakRefs and FinalizationRegistry API. WebAssembly based projects often rely on this API for wasm memory cleanup
```

Comment:
Should we add a compatibility date as well (I think we should?)? How exactly do we decide what goes behind a date vs flag vs both?

Reply:
If the intention is for `FinalizationRegistry` to just be available everywhere then yes, this will need a date where it is default switched on. We exclude the date for any flags that should always be explicitly opt-in.